packages/botonic-nlu/src/utils.js

```diff
@@ -0,0 +1,66 @@
+import { LANG_FLAG } from './constants'
+function escapeRegExp(str) {
+  return str.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, '\\$1')
+}
+
+export function replaceAll(str, find, replace) {
+  return str.replace(new RegExp(escapeRegExp(find), 'g'), replace)
+}
+
+export function shuffle(obj1, obj2) {
+  // shuffle two related arrays preserving indexes
+  var index = obj1.length
+  var rnd, tmp1, tmp2
+
+  while (index) {
+    rnd = Math.floor(Math.random() * index)
+    index -= 1
+    tmp1 = obj1[index]
```

Comment:
you can use https://stackoverflow.com/a/25910841/145289

Reply:
very cool!